Build fails on Zig 0.15.0-dev.1228

```
build.zig:36:33: error: no field named 'root_source_file' in struct 'Build.TestOptions'
    const tests = b.addTest(.{ .root_source_file = b.path("src/chameleon.zig") });
                                ^~~~~~~~~~~~~~~~
C:\Users\evany\scoop\apps\zig-dev\0.15.0-dev.1228\lib\std\Build.zig:778:25: note: struct declared here
pub const TestOptions = struct {
                        ^~~~~~
referenced by:
    runBuild__anon_32097: C:\Users\evany\scoop\apps\zig-dev\0.15.0-dev.1228\lib\std\Build.zig:2136:33
    main: C:\Users\evany\scoop\apps\zig-dev\0.15.0-dev.1228\lib\compiler\build_runner.zig:352:29
    4 reference(s) hidden; use '-freference-trace=6' to see all references
```

--- a/build.zig
+++ b/build.zig
@@ -33,7 +33,7 @@ pub fn build(b: *std.Build) void {
     const run_step = b.step("run", "Run the example");
     run_step.dependOn(&run_cmd.step);
 
-    const tests = b.addTest(.{ .root_source_file = b.path("src/chameleon.zig") });
+    const tests = b.addTest(.{ .root_module = cham_mod });
     const run_tests = b.addRunArtifact(tests);
     run_tests.step.dependOn(&tests.step);
     const run_tests_step = b.step("test", "Run tests");

--- a/build.zig.zon
+++ b/build.zig.zon
@@ -2,7 +2,7 @@
     .name = .chameleon,
     .version = "2.0.1",
     .fingerprint = 0x3f13cb6e8d5aec64, // Changing this has security and trust implications.
-    .minimum_zig_version = "0.15.0-dev.97+677b2d62e",
+    .minimum_zig_version = "0.15.0-dev.1228+6dbcc3bd5",
     .paths = .{
         "build.zig",
         "build.zig.zon",

